### PR TITLE
Adding sources.

### PR DESCRIPTION
**Pull request by victor-m-alvarez**

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -0,0 +1,868 @@
+# This file is automatically @generated by Cargo.
```
Comment (wngr) [inline Cargo.lock]: No `Cargo.lock` for libs.

File: build.rs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+// limitations under the License.
+extern crate cc;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // This produces an static library of lsm1.c that we can use to produce
+    // our Rust bindings.
+    cc::Build::new()
+        .file("src/lsm1/lsm1-ae2e7fc.c")
+        .define("LSM_MUTEX_PTHREADS", "1")
+        // .define("LSM_LOG_WORK", "1")
```
Comment (wngr) [inline build.rs]: ? Maybe leave a comment.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,98 @@
 # lsmlite-rs
-Rust bindings for SQLite’s lsm1 extension in stand-alone manner.
+
```
Comment (wngr) [inline README.md:2]: You might want to add some badges ~after publishing to crates.io and docs.rs~:
```
[![Cargo](https://img.shields.io/crates/v/lsmlite-rs.svg)](https://crates.io/crates/lsmlite-rs)
[![Documentation](https://docs.rs/lsmlite-rs/badge.svg)](https://docs.rs/lsmlite-rs)
```

**Review by wngr — APPROVED** [review]
💪

File: build.rs (hunk 2 above)
Comment (victor-m-alvarez) [reply build.rs]: Shouldn't be there. Removed.

File: Cargo.lock (hunk 1 above)
Comment (victor-m-alvarez) [reply Cargo.lock]: Done.

File: README.md (hunk 3 above)
Comment (victor-m-alvarez) [reply README.md:2]: Added. Thanks for letting me know.

**Pull request merged**